jobs/user-projects/general/nova-timetable.hcl

```diff
@@ -6,10 +6,14 @@ job "nova-timetable" {
     count = 1
 
     network {
-      port "db" {
+      port "redis" {
         to = 6379
       }
 
+      port "postgres" {
```

Comment:
I would calll this "db" rather than postgress to keep consistency with our other jobs